### DISCUSSION TITLE
webpack stuck at 99%

### DISCUSSION THREAD

**Discussion by jpvalery** [RyanClementsHax/storybook-addon-next#96] [Q&A]
Hi there,

Thanks for the great work on this. 

I was trying to do this on my own and definitely made it easier and more straightforward.

However, I'm running into the same issue than I was when trying to do from scratch. 
<img width="1294" alt="image" src="https://user-images.githubusercontent.com/4112343/174904814-083b58b0-aa35-4e2f-8cc7-9c024e82acb4.png"> 

(which seems to be the same issue than https://github.com/storybookjs/storybook/issues/14405). 

This time however I used your with-tailwind example as a base and just ran the command to init Storybook:
```
npx sb init --builder webpack5 -f
```

Any idea why? Do you encounter the same issue when running your example?

(Worth noting: build-storybook doesn't get stuck and properly generates the static site)

**Comment by RyanClementsHax** [comment]
Sorry I haven't. This is a little baffling to me actually. Does it still have this problem?
